build.rs

```diff
@@ -307,6 +378,7 @@ mod c {
         // also needs to satisfy intrinsics that jemalloc or C in general may
         // need, so include a few more that aren't typically needed by
         // LLVM/Rust.
+        #[allow(unexpected_cfgs)]
         if cfg!(feature = "rustbuild") {
             sources.extend(&[("__ffsdi2", "ffsdi2.c")]);
         }
```

Comment:
Could probably be removed. I don't think it's used anywhere in rust-lang/rust.
```suggestion
```

Reply:
I looked and couldn't find it being set in rust-lang/rust. Removed